General/OOD/SemanticConstructorOverloading/SemanticConstructorOverloading.Problem.Tests/Boxed Constructor Parameters/Good/SampleClass.Tests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Xunit;
+using FluentAssertions;
+using SemanticConstructorOverloading.Problem.Boxed_Constructor_Parameters.Good;
+
+namespace SemanticConstructorOverloading.Problem.Tests
+{
+    public class GoodBoxedStringObjectParametersTests
+    {
+        private readonly ReferenceClass defaultValues = new ReferenceClass("Lukewarm","Computer");
```

Comment:
Don't forget 3 essential test steps:
Arrange
Act
Assert

In your case:
Arrange is creating a data to compare to (expectation
Act is creating another object
Assert is comparing

So all good except placing all instances in one place, they should be a part of the test case.